shared/common-adapters/dumb.native.js

```diff
@@ -3,12 +3,10 @@
 import React from 'react'
 
 import Checkbox from './checkbox'
-import {TabBar, Text, Box, ListItem, Button} from './index'
```

Comment:
There are and we could enforce ordered imports and props and methods and
etc. I'm a fan but figure that might be too anal

On Friday, July 15, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/common-adapters/dumb.native.js
> <https://github.com/keybase/client/pull/3494#discussion_r71047514>:
>
> > @@ -3,12 +3,10 @@
> >  import React from 'react'
> >
> >  import Checkbox from './checkbox'
> > -import {TabBar, Text, Box, ListItem, Button} from './index'
>
> Is there an eslint plugin that orders these alphabetically?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3494/files/7f390a5dbec7d2f169db7b7dca29f9fa9af81e75#r71047514>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfhU0-13XhHqKgC5otgDxsyRdWBL5Q_ks5qWASXgaJpZM4JNxUM>
> .
>